### PR TITLE
docs: update companies and twitter handle

### PR DESCRIPTION
**Pull request by fzipi**

### CODE REVIEW

File: content/docs/tutorials/introduction.md [hunk 1]
```diff
@@ -157,10 +156,13 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 ### Companies using Coraza
 
 * [Babiel](https://babiel.com) (supporter)
```
Comment (jcchavezs) [inline content/docs/tutorials/introduction.md:158]: Is this still true?

Comment (jptosso) [reply content/docs/tutorials/introduction.md:158]: Nope, let's limit the list to the formal stuff we know are using coraza. Do we need permissions from companies or we should just list them ?

Comment (fzipi) [reply content/docs/tutorials/introduction.md:158]: To list then we don't need permissions. We can link to documentation or other stuff we deem necessary.

Comment (fzipi) [inline content/docs/tutorials/introduction.md:158]:
```suggestion
```

File: content/docs/tutorials/introduction.md [hunk 2]
```diff
@@ -156,11 +155,13 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 ### Companies using Coraza
 
-* [Babiel](https://babiel.com) (supporter)
+* [Traefik](https://owasp.org/blog/2024/03/19/traefik_owasp)
+* [United Security Providers AG](https://www.united-security-providers.ch/)
+* [Ambassador Labs](https://www.getambassador.io/docs/edge-stack/latest/howtos/web-application-firewalls)
```
Comment (fzipi) [inline content/docs/tutorials/introduction.md:160]:
```suggestion
* [Ambassador Labs](https://www.getambassador.io/docs/edge-stack/latest/howtos/web-application-firewalls)
* [Apache APISIX](https://apisix.apache.org/blog/2023/09/08/APISIX-integrates-with-Coraza/)
```

File: content/docs/tutorials/introduction.md [hunk 3]
```diff
@@ -156,11 +155,14 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
...
+* [Apache APISIX](https://apisix.apache.org/blog/2023/09/08/APISIX-integrates-with-Coraza/)
```
Comment (fzipi) [inline content/docs/tutorials/introduction.md:161]:
```suggestion
* [Apache APISIX](https://apisix.apache.org/blog/2023/09/08/APISIX-integrates-with-Coraza/)
* [Wallarm API Firewall](https://github.com/wallarm/api-firewall)
```

File: content/docs/tutorials/introduction.md [hunk 4]
```diff
@@ -156,11 +155,15 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 ### Companies using Coraza
```
Comment (fzipi) [inline content/docs/tutorials/introduction.md:156]:
```suggestion
### Companies/Products using Coraza
```

File: content/docs/tutorials/introduction.md [hunk 5]
```diff
@@ -154,13 +153,17 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 * Modsecurity team for creating ModSecurity
 * OWASP Coreruleset team for the CRS and their help
 
-### Companies using Coraza
+### Companies/Products using Coraza
```
Comment (jcchavezs) [inline content/docs/tutorials/introduction.md:156]: Maybe we want to add coraza cc @M4tteoP as per https://docs.tetrate.io/envoy-gateway/howto/coraza

**Pull request merged**